grumphp.yml

```diff
@@ -1,6 +1,6 @@
 grumphp:
   git_hook_variables:
-    EXEC_GRUMPHP_COMMAND: docker-compose run --rm -T php
+    EXEC_GRUMPHP_COMMAND: docker-compose run -e'PHP_ERROR_REPORTING=E_ALL & ~E_DEPRECATED' --rm -T php
```

Comment:
From my point of view I think deprecated errors must be fixed and not ignored, I've explained it detailedly here: https://github.com/Metadrop/drupal-boilerplate/pull/86#discussion_r1049655700